src/Extensions/Testing/AzureBlobStorageTests/Microsoft.Diagnostics.Monitoring.AzureBlobStorageTests.UnitTests.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Please move projects to `src/Tests` so that they can get the fix ups in the Directory.Build.props and Directory.Build.targets files.